app/projectsmodel.h

```diff
@@ -125,77 +125,84 @@ class ProjectsModel : public QAbstractListModel
     Q_INVOKABLE void fetchAnotherPage( const QString &searchExpression );
 
     //! Merges local and remote projects based on the model type
-    void mergeProjects( const MerginProjectsList &merginProjects, Transactions pendingProjects, bool keepPrevious = false );
+    void mergeProjects( const MerginProjectsList &merginProjects, bool keepPrevious = false );
```

Comment:
(optional) usually it is more readable instead of bool to use small enum with 2 options. so instead of keepPrevious = true/false have MergeProjectStrategy{KeepPrevious, DiscardPrevious(?)} so in callers you see what is done and not just `false`

Reply:
That is very cool, thanks, I added that